app/Services/FinanceService.php

```diff
@@ -19,6 +21,40 @@ public function __construct(
     ) {
     }
 
+    /**
+     * Add an expense, and properly tie it to a model, and know which
+     * airline this needs to be charged to
+     *
+     * @param array      $attrs      Array of attributes
+     * @param Model|null $model      The model this expense is tied to
+     * @param int|null   $airline_id The airline this expense should get charged to
+     *
+     * @return Expense
+     */
+    public function addExpense(array $attrs, Model $model = null, int $airline_id = null): Expense
+    {
```

Comment:
Maybe you should validate `$attrs` here, considering that when you call this function, you pass `$request->post()` or `$request->all()`, and it's not always a good idea to trust what the user may have entered and put it directly into the database.

Reply:
The model rules should validate it but let me double check that

Reply:
Oh cool, looks like the auto style fixes worked

Reply:
Yes, actually, it works only on branches of your repository, not on the PRs, I think. It works for my PRs because I also added it to my fork, and for your PRs because they are branches of your own repository.